src/components/StockTicker.js

```diff
@@ -2,13 +2,14 @@ import React from 'react';
 
 
 const formatColour = (value) => value > 0 ? 'green' : 'red'
-const arrowSign = (value) => value > 0 ? 'arrowUp' :'arrowDown'
+const arrowSign = (value) => value > 0 ? 'arrowUp' : 'arrowDown'
```

Comment:
`formatColour` and `arrowSign` could be condensed to the same function:

```
const getSign = value => value > 0 ? 'positive' : 'negative';
```
Then change your classnames to `color-positive`/`color-negative` and `arrow-positive`/`arrow-negative`. (which is also more declarative)